shared/pubsub.js

```diff
@@ -224,7 +238,7 @@ const defaultClientEventHandlers = {
   error (event: Event) {
     const client = this
 
-    console.error('[pubsub] Event: error', event)
+    console.log('[pubsub] Event: error', event)
```

Comment:
errors should be logged with `.error`

Reply:
It's not a programming error, only a so called "error" event. It happens e.g. when connectivity is lost or during the reconnection process when it takes more than one attempt. There is usually nothing to fix in such cases.

Meanwhile, our past Slack conversations regarding the break-after-sleep issue made me realize that this event could easily look more important than it actually is, when logged as an error. So I've been thinking about reducing the log level of these events.

Reply:
@snowteamer ok, then that needs to be explained in a comment here